### PR TITLE
fix(docs): update links, fix codeblocks, rephrase

### PR DESCRIPTION
**Pull request by ndom91**
## ☕️ Reasoning

- Clean up express documentation

## 🧢 Checklist

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged

## 📌 Resources

- [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md)
- [Contributing guidelines](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md)
- [Code of conduct](https://github.com/nextauthjs/.github/blob/main/CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

File: docs/docs/reference/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Here are the _state_ of planned and released packages under the `@auth/*` and `@
 | `@auth/core`           | Released (experimental).                                                                                                                                                               |
 | `@auth/sveltekit`      | Released (experimental, [help needed](#help-needed)).                                                                                                                                  |
 | `@auth/solid-start`    | Released (experimental, [help needed](#help-needed)). Community package: [`@solid-mediakit/auth`](https://www.npmjs.com/package/@solid-mediakit/auth)                                  |
-| `@auth/express`        | Released (experimental, [help needed](#help-needed)).                                                                                                                         |
+| `@auth/express`        | Released (experimental, [help needed](#help-needed)). See [docs](/reference/express)                                                                                                    |
```
Comment (balazsorban44) [inline docs/docs/reference/index.md:20]: Can you do the same for core, sveltekit, solid-start too?

**Review by balazsorban44 — CHANGES_REQUESTED** [review]

File: docs/docs/reference/index.md [hunk 2]
```diff
@@ -14,13 +14,13 @@ Here are the _state_ of planned and released packages under the `@auth/*` and `@
...
-| `@auth/astro`          | Planned, [help needed](#help-needed).                                                                                                                                                  |
-| `@auth/nuxt`           | Planned, [help needed](#help-needed). Community packages: [`@sidebase/nuxt-auth`](https://github.com/sidebase/nuxt-auth), [`@hebilicious/authjs-nuxt`](https://authjs-nuxt.pages.dev/) |
+| `@auth/core`           | Released (experimental). See [docs](/reference/core)                                                                                                                                   |
+| `@auth/sveltekit`      | Released (experimental). See [docs](/reference/sveltekit)                                                                                                                              |
+| `@auth/solid-start`    | Released (experimental). See [docs](/reference/sveltekit) Community package: [`@solid-mediakit/auth`](https://www.npmjs.com/package/@solid-mediakit/auth)                              |
```
Comment (balazsorban44) [inline docs/docs/reference/index.md:19]:
```suggestion
| `@auth/solid-start`    | Released (experimental). See [docs](/reference/solidstart) Community package: [`@solid-mediakit/auth`](https://www.npmjs.com/package/@solid-mediakit/auth)                              |
```

**Review by balazsorban44 — APPROVED** [review]

**Comment by binaryartifex** [comment]
getting a 404 page after clicking on the express auth link in the [docs](https://authjs.dev/getting-started/introduction)...

**Comment by ndom91** [comment]
> getting a 404 page after clicking on the express auth link in the [docs](https://authjs.dev/getting-started/introduction)...

Thats for the example app for which the PR to add it is still open. Coming soon though 🤞 

Docs can be found here: https://authjs.dev/reference/express

**Pull request merged**